<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_task_dialog</class>
 <widget class="QDialog" name="Add_task_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="task_type">
     <item>
      <property name="text">
       <string>Выберите тип задания</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Задание для проверки учителем</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Задание в Системе Сени</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>Название задания</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="title"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="text_task"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListWidget" name="task_list"/>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="files"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="file_label">
     <property name="text">
      <string>Расширения</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Add_task_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Add_task_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
